product/interfaces/templatefunctions/findProducts.go

```diff
@@ -33,27 +34,29 @@ func (tf *FindProducts) Func(ctx context.Context) interface{} {
 
 	/*
 		widgetName - used to namespace widget - in case we need pagination
-		config - map with certain keys - used to specifiy th searchRequest better
+		config - map with certain keys - used to specify the searchRequest better
 	*/
-	return func(namespace string, configs ...map[string]string) *application.SearchResult {
-		var searchConfig, keyValueFilters, filterConstrains map[string]string
+	return func(namespace string, configs ...pugjs.Map) *application.SearchResult {
+		var searchConfig = make(map[string]string)
+		var filterConstrains = make(map[string]string)
+		var keyValueFilters = make(map[string][]string)
 
```

Comment:
To init you can simply put:
searchConfig := make(map[string]string)
etc.
You do not need the var